blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxShardStateIO.java

```diff
@@ -53,19 +53,24 @@
         try {
             Select.Where statement = QueryBuilder
                     .select()
-                    .all()
+                    .column("key")
+                    .column("column1")
+                    .column("value")
+                    .writeTime("value")
```

Comment:
wow, nice that it's so easy! Either @usnavi or you will get merge conflict on this file. :-)